typings/single-spa.test-d.ts

```diff
@@ -3,6 +3,9 @@ import {
   mountRootParcel,
   registerApplication,
   pathToActiveWhen,
+  SingleSpaCustomEventDetail,
+  SingleSpaNewAppStatus,
+  SingleSpaAppsByNewStatus,
```

Comment:
I'm not sure how these are being imported, since I don't see them being exported above. Do you know why they're importable here? The tests passed, which I didn't expect.

Reply:
Apparently this is "by design" - https://github.com/microsoft/TypeScript/issues/38592

Though I didn't notice I hadn't explicitly exported them --- I just used code completion in VS Code when doing the import in `single-spa.test-d.ts`, which pulled them right in.

Reply:
Weird.